pom.xml

```diff
@@ -309,6 +327,41 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+          <id>integration</id>
+          <build>
+            <plugins>
+              <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven.surefire.version}</version>
+                <configuration>
+                    <runOrder>random</runOrder>
+                    <excludes>
+                      <exclude>**/*</exclude>
+                    </excludes>
```

Comment:
Why do we exclude all? In such case does anything happen?

Reply:
The idea is to use two different maven plugins:
- surefire - only for standard unit tests
- failsafe - only for integration tests
both of them are executed on maven phase "verify"
In this way we will exclude surefire (and all unit tests) when we want to run profile "integration" that is dedicated for integration tests only

Reply:
Does it mean that failsafe plugin won't fail if some of integration tests fail? I found ["integration-test" phase](http://webcache.googleusercontent.com/search?q=cache:w23PNt6FgikJ:https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html&hl=en&gl=de&strip=1&vwsrc=0) in maven